azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -209,6 +209,33 @@ func resourceArmKubernetesCluster() *schema.Resource {
 				Set: resourceAzureRMKubernetesClusterServicePrincipalProfileHash,
 			},
 
+			"addon_profile": {
```

Comment:
can we document these new fields in the docs? this file's located at `./website/docs/r/kubernetes_cluster.html`

Reply:
can we also add an acceptance test covering these new fields?